lib/msal-browser/src/app/Configuration.ts

```diff
@@ -51,44 +37,28 @@ export type TelemetryOptions = {
  * - tokenRenewalOffsetSeconds    - sets the window of offset needed to renew the token before expiry
  * - navigateFrameWait            - sets the wait time for hidden iFrame navigation
  */
-export type SystemOptions = {
-    loggerOptions?: BrowserLoggerOptions;
+export type BrowserSystemOptions = SystemOptions & {
+    loggerOptions?: LoggerOptions;
     networkClient?: INetworkModule;
-    loadFrameTimeout?: number;
-    tokenRenewalOffsetSeconds?: number;
-    navigateFrameWait?: number;
-    telemetry?: TelemetryOptions
-};
-
-/**
- * Logger options
- * 
- * - piiLoggingEnabled          - Used to configure whether piiLogging is enabled. Defaults to false.
- * - loggerCallback             - Callback for logger that determines how message is broadcast. Defaults to console.
- */
-export type BrowserLoggerOptions = {
-    piiLoggingEnabled?: boolean,
-    loggerCallback?: ILoggerCallback
+    popupWindowTimeout?: number;
```

Comment:
I might call this `windowHashTimeout` or something, to make it more clear what this timeout is specifically for.

Reply:
And then have the default value be something like `DEFAULT_WINDOW_HASH_TIMEOUT_MS`

Reply:
Changed the name of the variable. The constant is changed in a later PR